### DISCUSSION TITLE
Code rework idea: Card files

### DISCUSSION THREAD

**Discussion by Sup3rAw3som3Cod3r** [oskarrough/slaytheweb#182] [Ideas]
what if, instead of a single file for all cards, each card had it's own file, this way, it would be easier to add/change indivual cards

**Comment by oskarrough** [comment]
I can't remember when this was done, but it was! Thanks for the suggestion.

All the cards are defined in the [`src/content/cards` folder](https://github.com/oskarrough/slaytheweb/tree/main/src/content/cards).
